File: Udemy/Algoritmos e Logica de Programacao/Java/ExerciciosJavaEstruturaRepetitiva/src/application/ProblemaSequenciaImpares.java
package application;

import java.util.Locale;
import java.util.Scanner;

public class ProblemaSequenciaImpares {

	public static void main(String[] args) {
		/*
		 * Problema "sequencia_impares" (adaptado de URI 1067)
Leia um valor inteiro X. Em seguida mostre os ímpares de 1 até X, um valor por linha, inclusive o X,
se for o caso.
		 * */
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Digite o valor de X: ");
		int x = sc.nextInt();
		for(int i=0;i<x;i++) {
			if(i%2!=0) {
				System.out.println(i);
			}
		}

		sc.close();

	}

}

File: Algoritmos e Logica de Programacao/C++/Estrutura Repetitiva/ProblemaSequenciaImpares.cpp
/*
Problema "sequencia_impares" (adaptado de URI 1067)
Leia um valor inteiro X. Em seguida mostre os ímpares de 1 até X, um valor por linha, inclusive o X,
se for o caso.
*/

#include <iostream>
#include <iomanip>

using namespace std;

int main(){
	
	int x;
	cout << "Digite o valor de X: ";
	cin >> x;
	for(int i=0;i<x;i++){
		if(i%2!=0){
			cout << i << endl;
		}
	}

    return 0;
}
